package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "casbin",
-  "version": "2.0.3",
+  "version": "3.0.0-beta.1",
```

Comment:
Why the first commit is to change version number?

Reply:
Master maybe should is v3.x